pkg/apis/serving/v1alpha2/inferenceservice_validation.go

```diff
@@ -121,3 +129,18 @@ func validateCanaryTrafficPercent(spec InferenceServiceSpec) error {
 	}
 	return nil
 }
+
+const isvcNameFmt string = "[a-z]([-a-z0-9]*[a-z0-9])?"
+const isvcNameMaxLength int = 45
+
+var isvcRegexp = regexp.MustCompile("^" + isvcNameFmt + "$")
```

Comment:
Can you help move these to the const and var section on the top?

Reply:
I did that, Is this find?